src/app/course/load-more/load-more.component.less

```diff
@@ -0,0 +1,9 @@
+.load-more-container {
+    background-color: white;
+    height: 50px;
```

Comment:
You have been using rem in the application. Could you please use rem here as well ?